test/integration/common_defs.sh

```diff
@@ -1,7 +1,7 @@
 set -e
 CPODIR=/tmp/___cpo_integ_test
 rm -rf ${CPODIR}
-EPS=("tcp+k2rpc://0.0.0.0:10000" "tcp+k2rpc://0.0.0.0:10001" "tcp+k2rpc://0.0.0.0:10002")
```

Comment:
I need new nodes to create collection/schema with range partition to test query api. Current nodes are used by existing tests that uses hash partition.

Reply:
I don't think you need this. Other tests run on their own cluster

Reply:
I meant other tests in the same python file. They use collection with range partition 
https://github.com/futurewei-cloud/chogori-platform/blob/09605d89fbe5ea398e49d338a6f6f1e66f5b9901/test/integration/test_http.py#L45-L49
One option is I switch using range partition instead of hash partition for all tests. But then we don't be testing that it can create collection with hash partition.

Reply:
I see. You should probably use CollectionDrop then. We don't have this now in the proxy, but probably it's something to add next. For now, make an issue to add CollectionDrop support, and as part of that change we can have the tests use it

Reply:
Added https://github.com/futurewei-cloud/chogori-platform/issues/289 for CollectionDrop support